apps/docs/components/content/2.getting-started/vue.md

```diff
@@ -102,89 +100,80 @@ Finally, you'll need to add CSS directives to add each Tailwind layer to `src/st
 
 Now, you can import Storefront UI components in your app and all the Tailwind utilities from the `@storefront-ui/vue` library will be available in your project.
 
-
-
 <<<../../../../preview/nuxt/pages/showcases/Button/ButtonBlock.vue
 
-
-
 <Showcase showcase-name="Button/ButtonBlock" :show-source="false"/>
 
 #tab-2
 
 ## Nuxt 3
 
-If you prefer video guides, we have a quick video that can help you set up Storefront UI in your Nuxt 3 project.
-
-<iframe src="https://www.youtube-nocookie.com/embed/YamdPmZexto" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="allowfullscreen" class="w-full max-w-lg mx-auto rounded aspect-video relative mt-8 custom-block"></iframe>
-
 ### Install all dependencies
 
-With Nuxt 3, the fastest way to get started is to use the `@nuxtjs/tailwindcss` module. The [Nuxt Tailwind module](https://tailwindcss.nuxtjs.org/) will automatically install Tailwind CSS and PostCSS for you.
+With Nuxt 3, the fastest way to get started is to use the `@storefront-ui/nuxt` module. The [Storefront-io Nuxt module](https://www.npmjs.com/package/@storefront-ui/nuxt) will automatically install `@nuxtjs/tailwindcss` and storefront-ui tailwindcss presets.
 
-Additionally, you'll need to install the Storefront UI's Vue library and Tailwind preset.
+Additionally, you'll need to install the Storefront UI's Vue library and NuxtJs Tailwindcss Module.
 
 ```bash
 # npm
-npm i -D @nuxtjs/tailwindcss @storefront-ui/vue
+npm i -D @storefront-ui/nuxt @storefront-ui/vue @nuxtjs/tailwindcss
```

Comment:
Question: shouldn't `@nuxtjs/tailwindcss` be installed by our nuxt module?

Reply:
its installed only in nuxt  itself `modules` property in config, however we still need dependency

Reply:
dependency in our package.json, right? Not in the user codebase.
Like: `@nuxtjs/tailwindcss` does not require you to install `tailwindcss` itself

Reply:
dependency in client package.json. If user will install our `@storefront-ui/nuxt` module, which uses `@nuxtjs/tailwindcss` wont have it wont work. In other words `await installModule('@nuxtjs/tailwindcss');` in out module just adding  `@nuxtjs/tailwindcss` string to `nuxt.config.js` `modules` property. It does not installing dependency into package.json. I just double checked it.

Reply:
That's why it would be good to have `@nuxtjs/tailwindcss` as a subdependency of our module, right? IMO it would be a nice thing for the end users - they do not have to install/update `@nuxtjs/tailwindcss` unless they use it directly by themselves.

Now I'm wondering if we shouldn't do the same for `@storefront-ui/vue`. Do we need it to be explicitly installed by the user and not as a subdependency of `@storefront-ui/nuxt`?